package.json

```diff
@@ -35,7 +35,7 @@
     "serve-built-storybook": "npx http-server docs",
     "verify-node-version": "chmod +x ./scripts/verify-node-version.sh && ./scripts/verify-node-version.sh",
     "preversion": "node ./src/generateVersion.js",
-    "version": "npm run preversion && npm run verify-node-version && npm run build-storybook && git add -u ./docs && git add ./docs/* && git add ./src/version.ts && npm run compile",
+    "version": "npm run preversion && npm run verify-node-version && npm run build-storybook && git add -u ./docs && git add ./docs/* && git add ./src/version.ts && git add package.json",
```

Comment:
**Issue:** Whenever we bump the `package.json` version, it will make auto generated changes to `package-lock.json` whenever installing dependencies with `npm i`. I think we should also commit those auto generated changes.

For example, after manually updating package.json and running `npm i`:

![image](https://github.com/Constructor-io/constructorio-ui-autocomplete/assets/22061051/bb7ed734-52cc-4b63-858d-14bb17734b63)

So, I think we should:

- Run `npm i` to update the lock
- Also `git add package-lock.json`

Reply:
Oooh this is a real issue great catch will get this fixed up

Reply:
Turns out this is actually already handled by the `npm version` command along with the `package.json`. I removed this addition in response to the comment from @esezen